src/pages/home/Home.js

```diff
@@ -0,0 +1,74 @@
+import './Home.css';
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import HeaderContent from "../../components/headerContent/HeaderContent";
+import Tile from "../../components/tile/Tile";
+import Skeleton from "../../components/skeleton/Skeleton";
+
+function Home() {
+
+    const [getResult, setGetResult] = useState();
```

Comment:
Als je hier als default value een lege array hanteert, dan ben je je undefined check kwijt:
```suggestion
    const [getResult, setGetResult] = useState([]);
```